Templates/ServiceBusTopicTrigger-CSharp/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
+            "type": "serviceBusTrigger",
+            "direction": "in",
+            "topicName": "mytopic",
```

Comment:
Make your use of camel casing consistent throughout. In this case your topicName is not camel cased, but subscription is. Recommend checking the rest.